### ISSUE TITLE
Import PNG/JPEG/SVG 

### ISSUE THREAD

**Issue by sumtxt** [inkdropapp/inkdrop-import-html#1]
When importing HTML linked pictures are not imported unfortunately.

**Comment by craftzdog** [comment]
Right, it's intended because HTML files may include a lot of images such as clipped web pages.
Any workaround would be appreciated!

**Comment by sumtxt** [comment]
I understand, but maybe allow the user to choose if he wants to import images? In particular when importing my old Evernote notes that come with a few pictures, its a bit tedious to manually re-include those.  I can't code JS to actually provide some code.

**Comment by craftzdog** [comment]
Good point, what I just came up with is to check number of images and their file sizes.
I believe this would work well.

**Comment by craftzdog** [comment]
Moved to the forum since it's a core feature of Inkdrop: https://forum.inkdrop.info/t/support-importing-images/88
